src/python/grpcio_tests/setup.py

```diff
@@ -50,7 +50,8 @@
     'build_package_protos': grpc_tools.command.BuildPackageProtos,
     'build_py': commands.BuildPy,
     'run_interop': commands.RunInterop,
-    'test_lite': commands.TestLite
+    'test_lite': commands.TestLite,
+    'test_gevent': commands.TestGevent
```

Comment:
Always include the comma after the last element in multiline collection and dictionary literals.

Reply:
Fixed